src/kafka-util/src/bin/kgen.rs

```diff
@@ -78,7 +78,30 @@ fn main() -> anyhow::Result<()> {
             Arg::with_name("bootstrap")
                 .short("b")
                 .long("bootstrap-server")
-                .takes_value(true),
+                .takes_value(true)
+                .default_value("localhost:9092"),
+        )
+        .arg(
```

Comment:
I know you already started on this path but imo structopt is a way nicer interface to get args than Clap

Reply:
@benesch has already volunteered to rewrite it once we reach consensus on what opt parsing library to use. Personally, I have absolutely no opinion as long as all the features still work